### ISSUE TITLE
formal semantics

### ISSUE THREAD

**Issue by pdissaux** [saeaadl/BehaviorAnnex#5]
(this issue was referenced as G-04 in the [SaeAadlWiki](https://wiki.sei.cmu.edu/sae-aadl-subcommittee/index.php/Errata_from_the_Behavior_Annex))
Missing formal semantics for the Behavior Annex.
It should reflect the semantics of the core run-time and run-time services.

**Comment by jjhugues** [comment]
This seems like a long term task. Do we want to schedule this for any forthcoming document?

**Comment by smithdtyler** [comment]
> (this issue was referenced as G-04 in the [SaeAadlWiki](https://wiki.sei.cmu.edu/sae-aadl-subcommittee/index.php/Errata_from_the_Behavior_Annex))
> Missing formal semantics for the Behavior Annex.
> It should reflect the semantics of the core run-time and run-time services.

This link does not work for me - is it still valid?

**Comment by pdissaux** [comment]
Yes, unfortunately the AADL wiki that was hosted by SEI disapeared a few months ago 
I don't know about its contents 
The same for the main aadl.info web site. It now shows a SEI advert page instead.

**Comment by jjhugues** [comment]
Proposed resolution
- [ ] Target consistency with the core rather than a fully formal semantics. v3 could define this formal semantics
